spec/custom_matchers.rb

```diff
@@ -0,0 +1,12 @@
+RSpec::Matchers.define :belong_to do |expected|
+  match { |actual| (actual.to_a.uniq - expected.to_a.uniq).empty? }
+end
```

Comment:
I've added them both. And as expected, the response of Redis#scan is causing `nrows` to fail due to equality check. Maybe we can switch to range check?